cg/services/analysis_starter/configurator/file_creators/nextflow/sample_sheet/rnafusion.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants.constants import Strandedness
+from cg.io.csv import write_csv
+from cg.models.rnafusion.rnafusion import RnafusionSampleSheetEntry
+from cg.services.analysis_starter.configurator.file_creators.nextflow.sample_sheet.abstract import (
+    NextflowSampleSheetCreator,
+)
+from cg.store.models import Case, Sample
+from cg.store.store import Store
+
+
+class RNAFusionSampleSheetCreator(NextflowSampleSheetCreator):
+    def __init__(self, housekeeper_api: HousekeeperAPI, store: Store):
+        super().__init__(housekeeper_api)
+        self.store = store
+
+    def create(self, case_id: str, case_path: Path) -> None:
+        file_path = self.get_file_path(case_id=case_id, case_path=case_path)
+        content: list[list[str]] = self._get_content(case_id)
+        write_csv(file_path=file_path, content=content)
```

Comment:
If we move this function to the "abstract" class, would that simplify the mocking in the tests? We should double check if there is any trick for the rest of Nf pipelines but if not lets do it

Reply:
next PR!